contribs/gmf/apps/desktop_alt/Controller.js

```diff
@@ -33,6 +37,16 @@ if (!window.requestAnimationFrame) {
   window.location.href = 'https://geomapfish.org/';
 }
 
+module.value('gmfPermalinkOptions', /** @type {import('gmf/permalink/Permalink.js').PermalinkOptions} */ ({
```

Comment:
Move further below where `module` is declared. Can't compile otherwise.

Reply:
Fixed :-)